flutter/lib/src/event_processor/android_platform_exception_event_processor.dart

```diff
@@ -62,12 +64,41 @@ class AndroidPlatformExceptionEventProcessor implements EventProcessor {
       exception,
     );
 
-    return event.copyWith(
-      exceptions: [
-        ...?exceptions,
-        ...jvmException,
-      ],
-    );
+    final threads = _markDartThreadsAsNonCrashed(event.threads);
+
+    final jvmExceptions = jvmException.map((e) => e.key);
+
+    var jvmThreads = jvmException.map((e) => e.value).toList(growable: false);
+    if (jvmThreads.isNotEmpty) {
+      // filter potential duplicated threads
+      final first = jvmThreads.first;
+      jvmThreads = jvmThreads
+          .skip(1)
+          .where((element) => element.id != first.id)
+          .toList(growable: true);
+      jvmThreads.add(first);
+    }
+
+    return event.copyWith(exceptions: [
```

Comment:
If you enable `options.attachThreads`, how would that behave? Do we duplicate threads?

Reply:
I'll check